x/stakeibc/handler.go

```diff
@@ -62,3 +64,18 @@ func NewHandler(k keeper.Keeper) sdk.Handler {
 		}
 	}
 }
+
+// Handles stakeibc gov proposals
+func NewStakeibcProposalHandler(k keeper.Keeper) govtypes.Handler {
+	return func(ctx sdk.Context, content govtypes.Content) error {
+		switch c := content.(type) {
+		case *types.AddValidatorsProposal:
+			return k.AddValidatorsProposal(ctx, c)
```

Comment:
Why is AddValidatorsProposal new here?

Reply:
I consolidated`proposal_handler.go` and `handler.go` (and then removed `proposal_handler.go`)